pkg/configuration/configuration.go

```diff
@@ -222,11 +223,27 @@ func (ev *extendedViper) Set(key string, value interface{}) {
 	}
 }
 
+func bindEnvSnykErr(key string, err error) snyk_errors.Error {
+	return snyk_errors.Error{
+		Title:          "Unable to bind env",
+		Classification: "ACTIONABLE",
+		Level:          "warning",
+		Detail:         fmt.Sprintf("Unable to bind env: %v. Reason: %v", key, err),
+	}
+}
+
 func (ev *extendedViper) get(key string) interface{} {
 	ev.mutex.Lock()
 	defer ev.mutex.Unlock()
 
-	// try to lookup given key
+	// ensure we only get Snyk related keys (prefixed with SNYK_)
+	if ev.GetKeyType(key) == EnvVarKeyType {
+		err := ev.viper.BindEnv(key, strings.ToUpper(key))
```

Comment:
Suggestion: The docs for the function indicate that a single value does exactly what this code does, so let's keep the code clean and not add two params.